test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/address/MultipleDisposalsPropertyDetailsControllerSpec.scala

```diff
@@ -141,14 +148,19 @@ class MultipleDisposalsPropertyDetailsControllerSpec
 
         def test(
           draftReturn: DraftMultipleDisposalsReturn,
-          expectedBackLink: Call
+          expectedBackLink: Call,
+          userType: UserType
         ): Unit = {
           inSequence {
             mockAuthWithNoRetrievals()
             mockGetSession(
               SessionData.empty.copy(
+                userType = Some(userType),
                 journeyStatus = Some(
                   sample[FillingOutReturn].copy(
+                    subscribedDetails = sample[SubscribedDetails].copy(
```

Comment:
if it's an agent, for completeness it'd be good to set the `agentReferenceNumber` field in `SubscribedDetails` to `Some(sample[AgentReferenceNumber])` and to set it to `None` if it's not an agent